pkgs/top-level/all-packages.nix

```diff
@@ -157,11 +40,7 @@ let
   #
   # The result is `pkgs' where all the derivations depending on `foo'
   # will use the new version.
-  overridePackages = f:
-    let
-      newpkgs = pkgsFun newpkgs overrides;
-      overrides = mkOverrides pkgs (f newpkgs pkgs);
-    in newpkgs;
+  overridePackages = f: pkgsWithOverrides f;
```

Comment:
This replaces `packageOverrides` in config with something else. but sometimes one wants to chain overrides. I'd like to see a
```nix
let # for syntax coloring
  reoverridePackages = f: fix' (extend f self.__unfix__);
```